codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -14,6 +14,7 @@ class TermsAgreementInput:
     business_email: Optional[str] = None
     terms_agreement: bool = False
     marketing_consent: bool = False
+    customer_intent: str = None
```

Comment:
let's add optional

To be fair, i don't think this should ever be a blocker for onboarding, the page is a blocker for users from accessing the dashboard.